doc/classes/Dictionary.xml

```diff
@@ -294,6 +294,20 @@
 			<param index="1" name="overwrite" type="bool" default="false" />
 			<description>
 				Adds entries from [param dictionary] to this dictionary. By default, duplicate keys are not copied over, unless [param overwrite] is [code]true[/code].
+				[codeblock]
+				var dict = { "item": "sword", "quantity": 2 }
+				var other_dict = { "quantity": 15, "color": "silver" }
+
+				# Overwriting of existing keys is disabled by default.
+				# `Dictionary.merge()` modifies `dict` and does not return a value, so we need to use it on its own line.
```

Comment:
I don't follow this comment, if I'm honest.

Reply:
I think it's trying to explain that you can't do `print(dict.merge(other_dict))`

Reply:
Oh, that makes sense! Maybe it can be phrased better, but I would still remove it as excessive information. This is a very hypothetical sample anyway.